tests/run/special_methods_T561_py3.pyx

```diff
@@ -8,32 +8,9 @@ __doc__ = u"""
     >>> vs0 = VerySpecial(0)
     VS __init__ 0
 
-    >>> # Python 3 does not use __cmp__, so any provided __cmp__ method is
-    >>> # discarded under Python 3.
-    >>> vs0_cmp = vs0.__cmp__  # doctest: +ELLIPSIS
```

Comment:
The behaviour of these changes slightly now - they're no longer dropped completely but just added to the class dictionary (where presumably they'll do nothing). This seems a fairly inconsequential change but it would be possible to revert it if needed. I've just dropped the tests because I don't think we really care what happens to them.